torch/onnx/symbolic_opset9.py

```diff
@@ -1742,6 +1742,15 @@ def hardtanh(g, self, min_val, max_val):
     return g.op("Clip", self, min_f=min_val, max_f=max_val)
 
 
+@parse_args('v')
+def hardswish(g, self):
```

Comment:
Is it added for older opsets 7 and 8 (assuming they support all the needed ops)? If not then we should ensure there's a proper error message coming out for that (I think we already have a list for ops the are not supported in 7 and 8).

Reply:
Yes, the test cases for opset 7 and 8 are passing